docs/excel/excel-add-ins-ranges-string-match.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Find a string within a range using the Excel JavaScript API
```

Comment:
I'm not sure the "in a range" part is necessary. What do you think?
```suggestion
title: Find a string using the Excel JavaScript API
```

Reply:
I don't have a strong reason to keep it. Committing!